datalake/src/test/java/com/sequenceiq/datalake/flow/SdxReactorFlowManagerTest.java

```diff
@@ -147,6 +155,14 @@ void testTriggerCcmUpgradeFlow() {
         verify(reactor, times(1)).notify(eq(UPGRADE_CCM_UPGRADE_STACK_EVENT.event()), any(Event.class));
     }
 
+    @Test
+    void testTriggerSdxResizeEventSend() {
+        SdxCluster sdxCluster = getValidSdxCluster();
+        doNothing().when(eventSenderService).sendEventAndNotification(any(), anyString(), any(ResourceEvent.class));
```

Comment:
Please use argument matchers with explicit arguments whenever possible. The catch-all forms `any()` etc. are definitely permissible when used in a `verify(foo, never())` like construct, but may result in an overly lazy test logic in other cases. In particular, this should at least check for `ResourceEvent.DATALAKE_RECOVERY_STARTED` being passed in the 3rd arg.

By the way, `doNothing()` is the default behavior and is usually not needed. This call should rather be checked with a `verify(eventSenderService).sendEventAndNotification()` construct, just as done for `reactor.notify()` below.